ChangeLog

```diff
@@ -69,6 +69,7 @@ Changes
      Found by Coverity, reported and fixed by Peter Kolbus (Garmin). Fixes #2309.
    * Add test for minimal value of MBEDTLS_MPI_WINDOW_SIZE to all.sh.
      Contributed by Peter Kolbus (Garmin).
+   * Extended .gitignore to ignore Visual Studio artifacts. Fixed by ConfusedSushi.
```

Comment:
Minor: the changeLog entry should be last at the list, for correct history tracking, and for a script that merges the ChangeLog this way.
This can be done during gatekeeping
cc @Patater